website/content/partials/known-issues/1_15-audit-file-sighup-does-not-trigger-reload.mdx

```diff
@@ -0,0 +1,31 @@
+### File audit devices do not honor SIGHUP signal to reload
+
+#### Affected versions
+
+- 1.15.0
+
+#### Issue
+
+Sending [`SIGHUP`](/vault/docs/audit/file#log-file-rotation) to Vault after rotating
+an audit log file doesn't reopen the file path as expected.
+
+If the file was moved (renamed), then Vault will continue to write data to the location
+of the moved file.
+
+If the file was deleted, then Vault will continue to write data to the deleted file.
+The file will be deleted by the OS once Vault is sealed or restarted, but
+until then will continue to consume disk space due to audit entries being written.
```

Comment:
This is probably obvious, but: independent of the fact that space won't be reclaimed, should we also highlight the fact that audit records will be lost here?

Reply:
Yes, really good call. Thanks @ncabatoff 😄 I'll add an important note to try and draw attention to this.